examples/water_sampling_mc.py

```diff
@@ -71,6 +71,9 @@ def test_exchange():
     parser.add_argument("--iterations", type=int, help="Number of iterations", default=1000000)
     parser.add_argument("--equilibration_steps", type=int, help="Number of equilibration steps", default=50000)
     parser.add_argument("--seed", default=2024, type=int, help="Random seed")
+    parser.add_argument(
+        "--save_last_frame", type=str, help="Store last frame as a npz file, used to verify bitwise determinism"
```

Comment:
nit: could this be `action="store_true"` instead of `type=str`?

Reply:
This could be, the reason it is a string is to be able to write out names to disambiguate between the targeted and untargeted modes.

Reply:
sorry, disregard - I'd misread this as a boolean flag.